File: task-03/prime.py

def prime(n):
    count=0
    for i in range(1,n+1):
        if n%i==0:
            count+=1
    if count==2:
        return 1
    else:    
        return 0

print('Enter the limit: ')
n=int(input())
for i in range(2,n+1):
    if prime(i)==1:
        print(i,end=' ')
print('\n')

File: task-03/prime.rb


def prime(num)
    count = 0
    for i in 1..num
        if num % i == 0
            count += 1
        end
    end
    return count
end

puts "Enter the limit: "
n = gets.chomp.to_i
puts n

for i in 2..n+1
    if prime(i) == 2
        puts i
    end
end

